src/AppInstallerCommonCore/Public/winget/ManifestValidation.h

```diff
@@ -48,6 +48,9 @@ namespace AppInstaller::Manifest
         const char* const MissingManifestDependenciesNode = "Dependency not found: ";
         const char* const NoSuitableMinVersion = "No Suitable Minimum Version: ";
         const char* const FoundLoop = "Loop found.";
+        const char* const ExceededAppsAndFeaturesEntryLimit = "Only zero or one entry for Apps and Features can be specified for InstallerType portable.";

```

Comment:
I'm curious, why aren't these localized like the rest of the strings in winget (unless they are just too hard to translate)?

Reply:
We have a layering problem right now w/r/t localization.  Localization is done in CLICore, because it always has a loc file (packaged or colocated for unpackaged).  CommonCore is also used for the util binary, and we have not required that it carry around a loc file to function.

There are some changes coming that might end up requiring us to deal with this (aka move localization awareness into CommonCore), and at that point we would move these to be localized.  I think we will also need to create a compiled in backstop from the checked in .resw, so that we can continue to have the same functionality if the loc file isn't there.